src/redisearch_rs/redisearch_rs/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "redisearch_rs"
```

Comment:
A short description would be good. :)

Reply:
I have included it as a module-level comment in the crate root. Alternatively, we can have it as a README.md at the top-level. What do you prefer?